core/chains/evm/logpoller/log_poller.go

```diff
@@ -215,9 +215,9 @@ func (lp *logPoller) RegisterFilter(filter Filter) error {
 			// Nothing new in this filter
 			return nil
 		}
-		lp.lggr.Warnw("Updating existing filter %s with more events or addresses", "filter.Name", filter.Name)
+		lp.lggr.Warnw("Updating existing filter with more events or addresses", "filter.Name", filter.Name)
 	} else {
-		lp.lggr.Debugf("Creating new filter %s", filter.Name)
+		lp.lggr.Debugw("Creating new filter", "filter", filter)
 	}
```

Comment:
If this works, then we should probably do the same for the `Warnw` just before it.  (Use `filter` instead of `filter.Name`).  The new addresses and topics could be useful information.